### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by mgmr99** [Orchid-DOTNET/BscCSIT-2023#10] [General]
mail-address : ghimiremadhu2058@gmail.com
![error](https://user-images.githubusercontent.com/66813728/227193195-10c28fdf-875d-4788-b25e-219cfa609f5c.PNG)

**Comment by enigma037** [comment]
@mgmr99  let me add you again.

**Reply by enigma037** [reply]
I have added you again, accept the invitation.
One thing you can also try removing your credentials is 
Windows button-> Credential manager -> Windows Credentials
<img width="531" alt="image" src="https://user-images.githubusercontent.com/10417012/227215411-da18b06d-43f2-4f0c-b59b-82fe5fc26ced.png">

**Comment by enigma037** [comment]
@mgmr99 your assignment1 is reviewed. Assignment 2 is remaining.
